<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recommend_down</class>
 <widget class="QFrame" name="recommend_down">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="_1">
       <item>
        <widget class="QPushButton" name="_1_1">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/demo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_1_2">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/demo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_2">
       <item>
        <widget class="QPushButton" name="_2_1">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/demo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_2_2">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>270</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/demo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img/demo.qrc"/>
 </resources>
 <connections/>
</ui>
